docker/README.md

```diff
@@ -78,7 +78,7 @@ Example session:
     |   demo  | patroni3 | 172.22.0.4 |        | running |  1 |         0 |
     +---------+----------+------------+--------+---------+----+-----------+
 
-    postgres@patroni1:~$ etcdctl get --keys-only --prefix /service/demo
+    postgres@patroni1:~$ ETCDCTL_API=3 etcdctl get --keys-only --prefix /service/demo
```

Comment:
I would better add it to docker-compose.yml, similar to: https://github.com/zalando/patroni/blob/7370f70f13f2db968ed860ea9e8137d275dee860/docker-compose-citus.yml#L22

https://github.com/zalando/patroni/blob/7370f70f13f2db968ed860ea9e8137d275dee860/docker-compose-citus.yml#L56

Reply:
Yes, it would be better, but if I run the `ETCDCTL_API=3 etcdctl member list`
it doesn't show properly member's information about being **leader**, or peerURLs and clientURLs (looked for its flag , in json format i found the `peerURLs` and `clientURLs` but there is no `isLeader`)

- with `ETCDCTL_API=3`

```
3fec169331e2aba2, started, etcd1, http://etcd1:2380, http://172.31.0.6:2379
510b15c2cdc06ae5, started, etcd3, http://etcd3:2380, http://172.31.0.7:2379
76fc503b6c4957f4, started, etcd2, http://etcd2:2380, http://172.31.0.5:2379
```

- without `ETCDCTL_API=3` (its default API version is 2)

```
3fec169331e2aba2: name=etcd1 peerURLs=http://etcd1:2380 clientURLs=http://172.31.0.6:2379 isLeader=false
510b15c2cdc06ae5: name=etcd3 peerURLs=http://etcd3:2380 clientURLs=http://172.31.0.7:2379 isLeader=true
76fc503b6c4957f4: name=etcd2 peerURLs=http://etcd2:2380 clientURLs=http://172.31.0.5:2379 isLeader=false
```

Reply:
Well, for v3 API it indeed doesn't show the same member list output, but think about it like we already use 3.6 and v2 is fully deprecated and removed from Etcd.

Reply:
As im Seeing the default version is v2 ( cloned and did `docker compose up -d` )
so it would be the best if wee add the `ETCDCTL_API=3` in `docker/patroni.env`

```
postgres@patroni1:~$ etcdctl --version 
etcdctl version: 3.3.13
API version: 2
```

Reply:
Yes, please try adding it to Dockerfile and Dockerfile.citus and verify that it works.
Of course, after that they could be removed from docker-compose-citus.yml

Reply:
Thank You, I will do it.
What test case do you recommend to test and verify with that change should I consider?

Reply:
At least verify that the whole stack could be created with `docker-compose up` and that some basic commands from the readme work.

Reply:
- The `ETCDCTL_API=3` has been added to Dockerfile.
- Whole Stack created via `docker-compose up`, Working as well as before the change in Dockerfile
- `ETCDCTL_API=3` removed from `docker/README.md` from the previous PR
- couldn't apply the change to `Dockerfile.citus` because of failure in detection CA files , ...

Reply:
> couldn't apply the change to Dockerfile.citus because of failure in the detection of CA files, ...

I am not sure what you mean.

Reply:
- My bad , it was not the CA Files ,it was because of Filtering (Or Sanctions).
- It couldn't get GPG key for citus from `https://packagecloud.io/citusdata/community/gpgkey`properly (403 Error), I tried it in another server (Free Internet) and its okay.
- it is built with the `ENV ETCDCTL_API=3` in `Dockerfile.citus`  and the whole stack is up and running via `docker compose -f docker-compose-citus.yml up -d ` :

```
[root@Hetzner-Centos ~]# docker exec -it demo-coord2 etcdctl version 
etcdctl version: 3.3.13
API version: 3.3
```

whole Stack is up And Running: 

```
CONTAINER ID   IMAGE                    COMMAND                  CREATED         STATUS         PORTS                                                                                      NAMES
c20784bb023c   patroni-citus            "/bin/sh /entrypoint…"   8 minutes ago   Up 8 minutes                                                                                              demo-coord2
c272ba76e303   patroni-citus            "/bin/sh /entrypoint…"   8 minutes ago   Up 8 minutes                                                                                              demo-work1-2
d3f8be5db007   patroni-citus            "/bin/sh /entrypoint…"   8 minutes ago   Up 8 minutes                                                                                              demo-work2-2
0d14c070dba4   patroni-citus            "/bin/sh /entrypoint…"   8 minutes ago   Up 8 minutes                                                                                              demo-work1-1
a02aeb90ff1d   patroni-citus            "/bin/sh /entrypoint…"   8 minutes ago   Up 8 minutes                                                                                              demo-coord3
224579a77e5a   patroni-citus            "/bin/sh /entrypoint…"   8 minutes ago   Up 8 minutes                                                                                              demo-etcd1
4f28521e9f04   patroni-citus            "/bin/sh /entrypoint…"   8 minutes ago   Up 8 minutes                                                                                              demo-work2-1
a8ec5266daf3   patroni-citus            "/bin/sh /entrypoint…"   8 minutes ago   Up 8 minutes                                                                                              demo-etcd2
364bc4c7bece   patroni-citus            "/bin/sh /entrypoint…"   8 minutes ago   Up 8 minutes                                                                                              demo-coord1
fc6862ea2bf2   patroni-citus            "/bin/sh /entrypoint…"   8 minutes ago   Up 8 minutes                                                                                              demo-etcd3
5b4943064917   patroni-citus            "/bin/sh /entrypoint…"   8 minutes ago   Up 8 minutes   0.0.0.0:5000-5001->5000-5001/tcp, :::5000-5001->5000-5001/tcp                              demo-haproxy

```

Reply:
Cool, please don't forget to push these changes

Reply:
All Done.

- Add the `ENV ETCDCTL_API=3` in `Dockerfile.citus`
- Removed `ENV ETCDCTL_API=3` from `docker-compose-citus.yml`
- All `docker/README.md` Updated (Now it could get rid of the `outdated` tag :) )